### PR TITLE
chore: pull request template

### PR DESCRIPTION
**Pull request by Pankeking**

### CODE REVIEW

**Review by alcpereira — CHANGES_REQUESTED** [review]
As discussed in DM, the template should be a bit more complete and styled.

**Review by alcpereira — APPROVED** [review]

**Review by alcpereira — CHANGES_REQUESTED** [review]
Do you mind updating the branch to the last version we revised together please?

**Comment by Pankeking** [comment]
> Do you mind updating the branch to the last version we revised together please?

I didn't push it last night, it should be updated now

File: .env.example
Comment (alcpereira) [inline .env.example:1]: This change should be undone.

**Review by alcpereira — APPROVED** [review]
Great job 💪

File: .env.example
Comment (Pankeking) [reply .env.example:1]: what is an optimal way to undo a change?

**Review by Kenspectacle — CHANGES_REQUESTED** [review]
Looks about right to me so far!

Though I am not exactly sure what you meant by "Documentation is updated" in the Reviewer Checklist, maybe a slight more detail/rewording there?

**Comment by Pankeking** [comment]
> Looks about right to me so far!
> 
> Though I am not exactly sure what you meant by "Documentation is updated" in the Reviewer Checklist, maybe a slight more detail/rewording there?

"Documentation regarding the PR has been properly updated"
Is this clearer?
Any suggestion welcome

**Comment by Kenspectacle** [comment]
> > Looks about right to me so far!
> > Though I am not exactly sure what you meant by "Documentation is updated" in the Reviewer Checklist, maybe a slight more detail/rewording there?
> 
> "Documentation regarding the PR has been properly updated" Is this clearer? Any suggestion welcome

Nice, it makes much more sense to me! :)

**Review by Kenspectacle — APPROVED** [review]
LGTM :shipit:

**Pull request closed without merging**